poolakey/src/main/aidl/com/android/vending/billing/IInAppBillingService.aidl

```diff
@@ -34,4 +34,13 @@ interface IInAppBillingService {
 
     int consumePurchase(int apiVersion, String packageName, String purchaseToken);
 
+    Bundle getBuyIntentV2(
+        int apiVersion,
+        String packageName,
+        String sku,
+        String type,
+        String developerPayload
+    );
```

Comment:
Please use the Java wrapping style, just like the `getBuyIntent` function above.